### ISSUE TITLE
Asustor compatibility break

### ISSUE THREAD

**Issue by sandro09021982** [linuxserver/docker-sonarr#217]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Expected Behavior
I had Sonarr 3.0.8.1507-ls143 installed and it was stopping responding from time to time, forcing me to restart the container. I decided to update to the latest version available, 3.0.8.1507-ls145, to see if it fixed the problem.

## Current Behavior
The app didn't work. Mono keeps running, Docker accuses the container of running too but Sonarr is unavailable, I can't access the web interface. I went back to the version that was previously installed and then the app started working again.

## Steps to Reproduce

1.
2.
3.
4.

## Environment
**OS: ADM 4.0.5.RT42 (Asustor)
**CPU architecture: arm32
**How docker service was installed: nas OS provided

I have an Asustor AS1002T NAS and since the manufacturer stopped providing native apps for the system, I have been using Docker-CE and LinuxServer images: Deluge, Sonarr and Jackett. Some time ago I tried to install Bazarr but it had the same problem as the last version of Sonarr: Docker says it is running but it is not accessible. I ended up giving up using the Application because I couldn't get it to work. The same thing happened with Deluge when I tried to upgrade from version 2.0.3 to version 2.0.5. I went back to 2.0.3.

In short: some change was made to the image creation process or something else that broke compatibility with Asustor. Seems to be something with Alpine. For example, Deluge 2.0.3 uses Ubuntu as a base, while 2.0.5 apparently uses Alpine. In the case of Sonarr it could be something with Mono. The version that works uses Mono 5, the problematic one apparently 6.

## Command used to create docker container (run/create/compose/screenshot)
docker run -d --name=sonarr -e PUID=999 -e PGID=999 -e TZ=America/Sao_Paulo -p 8989:8989 -v /volume1/Docker/Sonarr/data:/config -v /volume1/Seriados_1:/tv1 -v /volume2/Seriados_2:/tv2 -v /volume1/Download/Completos:/downloads --restart unless-stopped lscr.io/linuxserver/sonarr:latest

**Comment by Roxedus** [comment]
Please follow the template.

**Comment by sandro09021982** [comment]
> Please follow the template.

Check if it's ok, please?!

**Comment by sandro09021982** [comment]
> Still doesnt have a log.

How do I get this log? Because the problem is that Sonarr installs but I can't interact with it because the web interface doesn't open.

**Comment by Xboarder56** [comment]
> sonarr

if the container starts you should be able to pull it via this from the CLI (ssh): 

docker logs sonarr
